code/modules/defenses/sentry.dm

```diff
@@ -107,6 +112,8 @@
 
 /obj/structure/machinery/defenses/sentry/get_examine_text(mob/user)
 	. = ..()
+	if(has_lifespan == TRUE)
```

Comment:
Same as above. This examine text can also be put into the subclasses get_examine_text